README.md

```diff
@@ -53,3 +53,81 @@ USE SCHEMA your_default_dbt_schema
 It's important to note that removing a particular query does not guarantee your Snowflake bill to decrease by the cost associated with that query. Snowflake bills based on the number of seconds your warehouse is running. If other queries are running at the same time as the query you removed, you will still be billed for that time.
 
 For the cloud services credits associated with your queries, you are only billed for them if they exceed 10% of your compute credit consumption on a given day. The cost per query model correctly handles this nuance by performing this applicability check prior to adding any cloud service costs to the queries. However, for queries that ran on the current date, this calculation may be innacurate. For example, say the cost per query model runs at 10AM today. For all the queries that occured before 10AM today, it finds that the total cloud service credits consumed was 15, and the total compute credits consumed was 100. Since this is greater than 10%, 5 cloud services credits will be allocated to the queries from today. It's possible that this fraction shifts as more queries run, and at the end of the day we have 90 cloud service credits and 1000 compute credits. If we re-run the model again after the day is completed, the queries were no longer have any cloud service costs associated with them. In the grand scheme of things, this won't make any significant difference in your analysis, but it is worth calling out.
+
+## Contributing
+
+### Initial setup
+
+1. Install pipx
+```bash
+pip install pipx
+pipx ensurepath
+```
+
+2. Install tox
+```bash
+pipx install tox
+```
+
+3. Install pre-commit
+```bash
+pipx install pre-commit
+```
+
+4. Configure your profile (follow the prompts)
+```
+dbt init
+```
```

Comment:
this creates a new `dbt-project.yml` right? do we want that?

Reply:
When run inside an existing project it just configures `~/.dbt/profiles.yml`.

Reply:
ah amazing!